rust-runtime/aws-smithy-runtime-api/src/client/orchestrator.rs

```diff
@@ -32,6 +34,62 @@ pub type BoxError = Box<dyn std::error::Error + Send + Sync + 'static>;
 pub type BoxFuture<T> = Pin<Box<dyn StdFuture<Output = Result<T, BoxError>> + Send>>;
 pub type Future<T> = NowOrLater<Result<T, BoxError>, BoxFuture<T>>;
 
+/// Configuration for how default protocol headers are serialized
+#[derive(Clone, Debug, Default)]
+pub struct SerializeDefaultHeaders {
```

Comment:
Should move this to inlineables so that it can be `pub(crate)`, and then rename to something like `HeaderSerializationSettings`.